api/src/main/java/org/openmrs/module/mohbilling/utils/Utils.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.mohbilling.utils;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
```

Comment:
This is what we recommend for logging: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging